File: Samples-Java/Samples/src/main/java/activities/DataFormExamples.java
package activities;

import java.util.ArrayList;
import java.util.LinkedHashMap;

import fragments.dataform.*;

public class DataFormExamples implements ExamplesProvider {
    @Override
    public String controlName() {
        return "DataForm";
    }

    @Override
    public LinkedHashMap<String, ArrayList<ExampleFragment>> examples() {
        LinkedHashMap<String, ArrayList<ExampleFragment>> dataFormExamples = new LinkedHashMap<>();

        ArrayList<ExampleFragment> result = new ArrayList<>();
        result.add(new DataFormGettingStartedFragment());
        result.add(new DataFormFeaturesFragment());
        result.add(new DataFormEditorsFragment());
        result.add(new DataFormAutoCompleteTextViewEditorFragment());
        result.add(new DataFormValidationFragment());

        result.add(new DataFormGroupLayoutFragment());
        result.add(new DataFormExpandableGroupsFragment());
        result.add(new DataFormPlaceholderLayoutFragment());
        result.add(new DataFormValidationBehaviorFragment());
        result.add(new DataFormValidationModeFragment());

        result.add(new DataFormJsonEditFragment());
        result.add(new DataFormSchemaSetupFragment());

        result.add(new DataFormCommitEventsFragment());

        result.add(new DataFormEditorStylesFragment());
        result.add(new DataFormLabelPositionsFragment());
        result.add(new DataFormImageLabelsFragment());

        dataFormExamples.put("Features", result);

        return dataFormExamples;
    }
}

File: Samples-Xamarin/Adapters/DataFormExamples.cs
using System;
using Android.Widget;
using Java.Util;
using Android.Views;

namespace Samples
{
	public class DataFormExamples : Java.Lang.Object, ExamplesProvider
	{
		private LinkedHashMap dataFormExamples;

		public DataFormExamples(){
			this.dataFormExamples = this.getDataFormExamples();
		}

		public String ControlName() {
			return "Data Form";
		}

		public LinkedHashMap Examples(){
			return this.dataFormExamples;
		}

		private LinkedHashMap getDataFormExamples(){
			LinkedHashMap dataFormExamples = new LinkedHashMap();
			ArrayList result = new ArrayList();

			result.Add (new DataFormGettingStartedFragment());
			result.Add (new DataFormFeaturesFragment ());
			result.Add (new DataFormEditorsFragment ());
			result.Add (new DataFormValidationFragment ());
			result.Add (new DataFormGroupLayoutFragment ());
			result.Add (new DataFormPlaceholderLayoutFragment ());
			result.Add (new DataFormValidationBehaviorFragment ());
			result.Add (new DataFormValidationModeFragment ());
			result.Add (new DataFormJsonEditFragment ());
			result.Add (new DataFormSchemaSetupFragment ());

			result.Add (new DataFormCommitEventsFragment());

			result.Add (new DataFormEditorStylesFragment());
			result.Add (new DataFormLabelPositionsFragment());

			dataFormExamples.Put ("Features", result);

			return dataFormExamples;
		}
	}
}

